app/src/main/res/values/attrs.xml

```diff
@@ -2,11 +2,16 @@
 <resources>
     <declare-styleable name="NestedScrollableHost">
         <attr name="scrollDirection" format="enum">
-            <enum name="both" value="0"/>
-            <enum name="vertical" value="1"/>
-            <enum name="horizontal" value="2"/>
+            <enum name="both" value="0" />
+            <enum name="vertical" value="1" />
+            <enum name="horizontal" value="2" />
         </attr>
-        <attr name="preferHorizontal" format="integer"/>
-        <attr name="preferVertical" format="integer"/>
+        <attr name="preferHorizontal" format="integer" />
+        <attr name="preferVertical" format="integer" />
+    </declare-styleable>
+    <declare-styleable name="AppWidgetAttrs">
```

Comment:
This styleable is never mentioned anywhere else

Reply:
Hello @ByteHamster sir, this is my first code contribution so please help me to resolve these issue.

Reply:
Sorry, I cannot give a full tutorial on how to do the Pull Request. Doing so would take me more time than implementing it myself. Try running the app on your phone and experimenting with the changes until it works :)